src/include/buffer.h

```diff
@@ -1248,6 +1248,10 @@ struct error_code;
     uint32_t digest() {
       return crc;
     }
+
+    void operator=(const bufferhash &r) {
+      crc = r.crc;
+    }
```

Comment:
I think that we should introduce this as a separate commit with an explanation as this is a common change. Possibly, we could wrap it with `WITH_SEASTAR` even though it has no Classical users.

Reply:
done.

Reply:
removed buffer.h change.

Reply:
> removed buffer.h change.

Even better :)